src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+
+### Focus sur l'impact
+
+Ils comprennent l'importance de l'impact de leur travail sur le produit final et sur l'utilisateur/client. Ils sont motivés par la création de valeur réelle, plutôt que par la simple écriture de code.
+
+C’est un point auquel, je suis particulièrement sensible. Au début, on passe quasi tous par la phase “pisseurs de code” comme on dit dans le jargon. Personnellement, je ne la vois pas systématiquement que d’un mauvais œil, c’est une partie intégrante d’une phase de progression, l’erreur serait d’y rester pire de penser que plus de code impressionnant on envoi, plus on est excellente.
+
+Une phrase de devs seniors avec qui j’ai échangé et qui m’a beaucoup marqué est la suivante :
+
+> “La meilleure ligne de code est celle que l’on n'écrit pas.”
+
+### Pragmatisme
+
+Les développeurs du top 20% sont pragmatiques. Ils savent faire la part des choses entre perfection et productivité. Ils comprennent quand il est nécessaire de faire des compromis pour avancer plus rapidement sans sacrifier la qualité du produit.
+
+C’est certainement la caractéristique sur laquelle j’aurais à le plus fournir d’effort. Etant à l’extrême perfectionniste, je vois littéralement comment cela est handicapant au quotidien. Si vous voulez faire partir des meilleurs vous devrez entrainer ce “soft skill” qui changera votre vie d’entrepreneur, de collaborateur, de freelance, etc. Pensez y vraiment, c’est dur mais nécessaire pour faire partir de la crème.
+
+### Responsabilité
+
+Enfin, ils prennent la responsabilité de leur travail. Ils sont conscients que chaque ligne de code qu'ils écrivent peut avoir des conséquences, et ils s'efforcent de produire un travail de qualité pour minimiser les bugs et les problèmes à long terme.
+
+C’est un sujet qui rejoint la partie caractéristique technique du respect/maitrise des bonnes pratiques du métier mais avec une touche d’éthique qui fait réellement la différence.
+
+Si comme moi, vous êtes naturellement à la recherche de sens dans tout ce que vous faites. Vous vous y trouverez forcément. Et si ce n’est pas le cas, vous devriez sérieusement y songer. Votre vous du futur dira merci à celui du présent, et cela entrainera un cercle vertueux qui fera de vous un dev du top tier. Autant pour vous même (confiance en soi, fierté du travail bien fait, se sentir utile, etc) mais aussi pour le monde extérieur, pairs-utilisateurs-clients (reconnaissance, utilisateurs satisfaits, évitement du pire, etc).
+
+## Conclusion
+
+Je suis assez convaincu que si vous êtes un développeur, vous avez sûrement déjà entendu 80% des caractéristiques mentionnées et ça parait toujours aussi bateau et vague pour vous.
+
+Comme moi, vous voulez surement du concret, quelque chose à se mettre sous la dent (et pas n’importe quelle dent, hein, celle d’un dev : on en fait pas des plus acérés). Je vous ai donc préparer deux outils que vous pouvez mettre en pratique dès maintenant.
+
+### Outil 1 : je pense à un formulaire Tally
```

Comment:
Do you have ressources to share? You don't have to explain everything, especialy if someone else do it better :)

Reply:
I'll take the time to add more resources tomorrow as it's the end of the day.

One question I'm asking myself is whether it's necessary to have read the whole resource or just an overview, or whether a book that's often recommended but hasn't been read can do the trick. 
For my part, the majority of this article is a digest of what I've learned so far and the AI to help me complete it and make it more audible to the majority. I'd love to know the best way to deal with this.